pages/navigationMenu/NavigationMenuItemHandler.inc.php

```diff
@@ -94,12 +94,6 @@ function view($args, $request) {
 			$contextId = $context->getId();
 		}
 
-		// Ensure that if we're previewing, the current user is a manager or admin.
-		$roles = $this->getAuthorizedContextObject(ASSOC_TYPE_USER_ROLES);
-		if (count(array_intersect(array(ROLE_ID_MANAGER, ROLE_ID_SITE_ADMIN), $roles))==0) {
-			fatalError('The current user is not permitted to preview.');
```

Comment:
It looks like this change just removes the check entirely, correct? Shouldn't previewing rightly be limited to admins and managers?

Reply:
@asmecher the preview function has the check. The view function is for displaying the  custom page only.

Reply:
OK, thanks -- will merge.